libraries/python/pocketsizefund/event/event.py

```diff
@@ -0,0 +1,35 @@
+"""Contains the event-based publisher and subscriber resources."""
+
+from kafka import KafkaConsumer, KafkaProducer
```

Comment:
not necessary now, but in future we might want to consider AIOKafka library, same basic API, just on the event loop